src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHeader.cs

```diff
@@ -87,5 +87,55 @@ internal partial struct TarHeader
         // If the archive is GNU and the offset, longnames, unused, sparse, isextended and realsize
         // fields have data, we store it to avoid data loss, but we don't yet expose it publicly.
         internal byte[]? _gnuUnusedBytes;
+
+        internal TarHeader(TarEntryFormat format)
+            : this(format, string.Empty, 0, default, 0)
```

Comment:
Can this just be default values on the constructor on line 97?

If not, it would be good to name `0, default, 0` arguments here so it is readable what they align with.

Also `0` for the `TarEntryType` enum is a bit confusing. Would RegularFile be more readable?